index.html

```diff
@@ -0,0 +1,170 @@
+          <div class="insidemain flex-center">
+            <i class="fa-sharp fa-solid fa-book-open-reader"></i>
+            <h3>Lecture</h3>
+            <p>Listen to the speakers from various countries
+              about the messages of sharing and opening.</p>
+          </div>
+        </article>
+        <article class="bg-color list-prog1">
+          <div class="insidemain flex-center">
+            <i class="fa-brands fa-accusoft"></i>
+            <h3>CC Exhibition</h3>
+            <p>Appreciate various creations applying
+              CC license of artists, organized from Art Center</p>
+          </div>
+        </article>
+        <article class="bg-color list-prog1">
+          <div class="insidemain flex-center">
+            <i class="fa-regular fa-comments"></i>
+            <h3>Forum</h3>
+            <p>Have the time to share your thoughts
+              and opinions with experts for each topic.</p>
+          </div>
+        </article>
+        <article class="bg-color list-prog1">
+          <div class="insidemain flex-center">
+            <i class="fa-solid fa-code-branch"></i>
+            <h3>Workshop</h3>
+            <p>Try creating your own work using open
+              source license rather than just watching at it.</p>
+          </div>
+        </article>
+        <article class="bg-color list-prog1">
+          <div class="insidemain flex-center">
+            <i class="fa-solid fa-gears"></i>
+            <h3>CC Ignite</h3>
+            <p>get opportunities to network with CC
+              affiliates around the world, also after the summit.</p>
+          </div>
+        </article>
+      </div>
+      <button type="button" name="join" class="btn-join">Join the Program</button>
+      <a class="see-pro"><u>THE WHOLE PROGRAM</u></a>
+      </div>
+    </section>
+
+    <section class="speakers flex-center" id="speakers">
+      <h2>Featured Speakers</h2>
+      <hr class="line">
+      <div class="speakers1" id="contspeak">
+        <!-- This item now in java-script -->
+      </div>
+      <button type="button" name="join" class="btn">MORE <i class="fas fa-angle-down down-arrow"></i></button>
+    </section>
+
+    <section class="partner flex-center" id="partner">
+      <h2>Partner</h2>
+      <hr class="line">
+      <div class="partner-link flex-center">
+        <a href="#"><img src="assests/sponcers/Pfizer.svg.png" alt="pfz-img"></a>
+        <a href="#"><img src="assests/sponcers/disney.png" alt="disney-img"></a>
+        <a href="#"><img src="assests/sponcers/Rolex_logo.svg.png" alt="rolex-img"></a>
+      </div>
+    </section>
+  </main>
+  <footer class="contact flex-center">
+    <h2>Contact</h2>
+    <hr class="line">
+    <div class="foot-cont">
+      <img src="assests/logo/logo.png" alt="img">
+      <p>THE 95TH ACADEMY AWARDS | 2023
+        Dolby Theatre at Ovation Hollywood
+        Sunday, March 12, 2023
+        Honoring movies released in 2022</p>
+    </div>
```

Comment:
- [ ] kindly consider aligning well these elements ao as to meet the  provided design 
- [ ] your design 👇 
![7k](https://user-images.githubusercontent.com/60146030/225400645-8882f9c4-8107-4181-887c-1590d0024448.PNG)

- provided design 👇 
![8k](https://user-images.githubusercontent.com/60146030/225400801-dd8c2ca2-24cb-4cca-8958-7f359e407954.PNG)
 you can implement that by using of `flex box` in your design and increasing the margin of the container of those two elements also consider changing the background color to the provided one  and this will contribute to your design to meet the requirements